master/master.cfg

```diff
@@ -62,13 +66,35 @@ from custom.settings import Settings    # noqa: E402
 from custom.steps import Git, GitHub    # noqa: E402
 from custom.workers import get_workers  # noqa: E402
 
+
+def set_up_sentry():
+    try:
+        release_sha = subprocess.check_output(["git", "rev-parse", "HEAD"], text=True)
+    except subprocess.SubprocessError:
+        log.err('WARNING: Could not get current SHA for the release!')
+        return
+
+    sentry_sdk.init(dsn=settings.sentry_dns, release=release_sha,
```

Comment:
Same comments :).  `dsn` or `dns`; you have both in different spots here.  And this should check whether `settings.get('sentry_d..', None)` is `None` and act accordingly, issuing a warning if it *is* `None` and doing the setup otherwise.

Reply:
Oh, sorry I totally missed that :( Thanks for the catch and the patience on the review!